controls/accordion/overview.md

```diff
@@ -8,14 +8,14 @@ slug: accordion-overview
 
 # .NET MAUI Accordion Overview
 
-**Telerik UI for .NET MAUI Accordion** is a vertically collapsible content panel that displays only one of its items at a time within the available space. RadAccordion helps you save screen space and at the same time present the content to the end user in an easily accessible way.
+**Telerik UI for .NET MAUI Accordion** is a vertically collapsible content panel that displays only one of its items at a time within the available space. `RadAccordion` helps you save screen space and at the same time present the content to the end user in an accessible way.
 
 ![.NET MAUI Accordion Overview](images/accordion-overview.png)
 
 ## Key features
 
-* [Collapsed/expanded states]({%slug accordion-expand-collapse-states%})&mdash;Accordion consists of AccordionItems that can host any content. The end users could show or hide this content by interacting with the headers of the control.
-* [Collapse All Items]({% slug accordion-expand-collapse-states%}#collapse-all-items)&mdash;You can allow the app users to fully collapse the Accordion through the CanCollapseAllItems boolean property.
+* [Collapsed/expanded states]({%slug accordion-expand-collapse-states%})&mdash;Accordion consists of AccordionItems that can present any content. The end users can show or hide this content by interacting with the headers of the control.
+* [Collapse All Items]({% slug accordion-expand-collapse-states%}#collapse-all-items)&mdash;You can allow the app users to collapse the Accordion through the `CanCollapseAllItems` boolean property.
```

Comment:
The term "Boolean" is typically written with a capital "B" because it is derived from the name of mathematician and logician George Boole.
```suggestion
* [Collapse All Items]({% slug accordion-expand-collapse-states%}#collapse-all-items)&mdash;You can allow the app users to collapse the Accordion through the `CanCollapseAllItems` Boolean property.
```